### PR TITLE
KUD-196 Fix issue loading the New Relic GraphQL extension

### PR DESCRIPTION
**Pull request by AlvinKudos**
Fix bug loading New Relic GraphQL extension.

See [draft release notes](https://github.com/KudosInc/node_logger/releases/tag/untagged-4cb6ac357055315d5241).

See [New Relic Albums service](https://rpm.newrelic.com/accounts/74807/applications/608249525/) for application logs, GraphQL operation name, distributed tracing.

### CODE REVIEW

**Comment by AlvinKudos** [comment]
Sample GraphQL operation names are `/query Albums__AlbumsService__ALL_ALBUMS_QUERY` and `/query no action`

**Comment by AlvinKudos** [comment]
Logs are decorated as expected, distributed tracing is turned on.

File: newrelicPlugin.js [hunk 1]
```diff
@@ -1,21 +1,14 @@
...
-const {
-  first,
-} = require('lodash/fp');
-
-const extraSpacesNewLineRemovalRegexp = new RegExp(/(\r\n|\n|\r|\s\s+)/gm);
-const QUERY_MUTATION_PATTERN = new RegExp(/query|mutation/);
-const QUERY_ACTION_PATTERN = new RegExp(/(?<=\{[ ]+)[A-Za-z0-9]+/);
+const helper = require('./helper');
 
 module.exports = () => {
```
Comment (talal7860) [inline newrelicPlugin.js:5]: Can we export the class directly, I don't see any reason of encapsulating it in a function

**Comment by talal7860** [comment]
@AlvinKudos did you get a chance to deploy this change to staging?

**Pull request merged**